.circleci/config.yml

```diff
@@ -208,6 +225,8 @@ jobs:
     environment:
       MAGENTO_DIR: magento
     steps:
+      - swissknife/trigger_pipeline:
```

Comment:
You shouldn't need to isntall on all jobs, only the job that triggers the other pipelines

Reply:
I think all of the unit_test_merchant_branch jobs trigger the pipeline, since they all call the tesT_unit_branch.sh script